docs/community_resources/release_process.md

```diff
@@ -0,0 +1,61 @@
+# eZ Platform Release Process and Roadmap
+
+## Release process
+
+eZ Platform is distributed in two flavors:
+
+- eZ Platform is an open source Content Management System (CMS) developed by eZ Systems together with the open source community. eZ Platform's code base is available on GitHub under the GPLv2 license. eZ Platform comes with no commercial support and maintenance services. It is supported by the community on public channels.
+- eZ Platform Enterprise Edition (EE) is commercial software available under eZ Business User License (BUL) to eZ Enterprise subscribers. It is comprised of eZ Platform Open Source edition, additional enterprise support and maintenance services, as well as additional features which are not available in the open source software.
+
+We manage the release of eZ Platform using an agile iterative process and a continuous software development model, which is why we provide two kinds of eZ Platform releases:
+
+- Long Term Support releases (LTS) which are supported by eZ Systems for a long period of time (see [support lifecycle below](#support-lifecycle)), for eZ Enterprise subscribers.
+- Fast Track releases (FT) give access to the latest features and are supported for a short period of time, only until the next FT release is introduced. These are supported for both the open source community and eZ Enterprise subscribers.
+
+FT releases are tailored for those who want to stay more on the bleeding edge, while LTS releases are suitable for highly stable enterprise rollouts.
+
+We usually release eZ Platform four times a year following the seasons (Winter, Spring, Summer and Fall). This usually includes one LTS release and three FT releases.
+
+## Versioning conventions
+
+Both eZ Platform editions use [semantic versioning](http://semver.org/).  
+
+The version number of eZ Platform and all its internal components follows the semantic versioning conventions: vX.Y.Z.
+
+- Changes to X indicate a breaking change to the API, and developers know that they may need to implement a re-write of their code to continue using the API.
+- Y represents new features and functionality.
+- Z represents patches, bug fixes, smaller improvements to unblock users, etc.
+
+This way, distribution files of our product are as follows:
+
+- eZ Platform uses its own unique semantic version number: ezplatform-vX.Y.Z.tgz
+- eZ Platform Enterprise uses its semantic version number: ezplatformenterprise-vX.Y.Z.tgz
```

Comment:
- for eZ Platform: ezplatform-vX.Y.Z.tgz
- for eZ Platform Enterprise: ezplatformenterprise-vX.Y.Z.tgz